third_party/opa/data-and-policies/user-created-policy-3/sample_policies.rego

```diff
@@ -0,0 +1,76 @@
+package dataapi.authz
+
+verdict[output] {
+	count(rule) == 0
+	output = {"action": {"name":"DenyAccess"}, "policy": "Deny by default"}
+}
+verdict[output] {
+	count(rule) > 0
+	output = rule[_]
+}
+rule[{"action": {"name":"RedactColumn", "columns": column_names}, "policy": description}] {
+	description := "Columns with Confidential tag to be redacted before read action"
+    #user context and access type check
+	input.action.actionType == "read"
+    input.context.intent == "Fraud Detection"
+	input.context.role == "Data Scientist"
+	input.resource.tags.residency == "Turkey"
+	input.action.processingLocation != "Turkey"
+	column_names := [input.resource.columns[i].name | input.resource.columns[i].tags.Confidential == "true"]
+}
+rule[{"action": {"name":"DenyAccess"}, "policy": description}] {
+	description := "Deny because the role is not Data Scientist when intent is Fraud Detection"
+    #user context and access type check
+    input.action.actionType == "read"
+	input.context.intent == "Fraud Detection"
+	input.context.role != "Data Scientist"
+	input.resource.tags.residency == "Turkey"
+}
+
+rule[{"action": {"name":"DenyAccess"}, "policy": description}] {
+	description = "Deny because columns have confidential tag"
+    #user context and access type check
+    input.action.actionType == "read"
+	input.context.intent == "Customer Behaviour Analysis"
+	input.context.role == "Business Analyst"
+	input.resource.tags.residency == "Turkey"
+    column_names := [input.resource.columns[i].name | input.resource.columns[i].tags.Confidential == "true"]
```

Comment:
Didn't we change it so tags could accept a boolean so no need for =="true"?

Reply:
Updated it in the rules.